crates/pixi_config/src/lib.rs

```diff
@@ -1153,6 +1183,63 @@ impl Config {
                     _ => return Err(err),
                 }
             }
+            key if key.starts_with("s3-options") => {
```

Comment:
Please add this to the `cli` documentation. And I noticed that it doesn't work with `pixi config set s3-options.bucked {JSON_VERSION_OF_THE_FULL_SET}` which would be a great addition.

Reply:
do you mean here? 
<img width="1081" alt="image" src="https://github.com/user-attachments/assets/f57e3e03-0abf-4971-9323-f22c35799346" />

the global config docs are already done in #2835.

![image](https://github.com/user-attachments/assets/ee891155-1705-42b2-b16c-ffac51937744)

is it okay if we leave this for the other pr?

Reply:
done in [`91f1585` (#2825)](https://github.com/prefix-dev/pixi/pull/2825/commits/91f15857343edda1da85c2f56826603ba8558630)

Reply:
Amazing, exactly what I wanted 💛